### DISCUSSION TITLE
如何限制连接桩只能连接一个边？

### DISCUSSION THREAD

**Discussion by G-Veigar** [antvis/X6#2290] [Q&A]
目前的思路是validateMagnet去校验，如果连接桩已经有链接了就false掉，validateConnection也同样处理。
问题是validateMagnet如何拿到是哪个连接桩触发的呢？

**Comment by pjwok** [comment]
感觉这个需求用这个配置项可以解决 [allowMulti](https://x6.antv.vision/zh/docs/tutorial/basic/interacting#allowmulti)
